clients/js/src/doc.ts

```diff
@@ -39,8 +40,8 @@ async function getHelpText(name: string, cmd: any): Promise<string> {
   ];
 
   const helpOutputs: Buffer[] = [];
-  for (const [name, cmd] of Object.entries(cmds)) {
-    const helpText = await getHelpText(name, cmd);
+  for (const cmd of Object.values(cmds)) {
```

Comment:
wouldn't
```suggestion
  for (const cmd of cmds) {
```
achieve the same thing?

Reply:
yep! just changed locally, thanks!